management/client.go

```diff
@@ -138,6 +138,14 @@ type Client struct {
 
 	// RequireProofOfPossession Makes the use of Proof-of-Possession mandatory for this client (default: false).
 	RequireProofOfPossession *bool `json:"require_proof_of_possession,omitempty"`
+
+	DefaultOrganization *ClientDefaultOrganization `json:"default_organization,omitempty"`
```

Comment:
We have to add a test case for testing this in `client_test.go`  because this relates to the Client Manager.